gettsim/taxes/zu_verst_eink/freibetraege.py

```diff
@@ -205,74 +205,83 @@ def eink_st_altersfreib_ab_2005(
     return out
 
 
-def eink_st_sonderausgaben_bis_2011(
-    kind: bool,
+def eink_st_sonderausgaben_bis_2011_tu(
     eink_st_abzuege_params: dict,
+    anz_erwachsene_tu: int,
 ) -> float:
     """Calculating individual sonderausgaben for childcare until 2011.
 
     There is only a lumpsum payment implemented.
     Parameters
     ----------
-    kind
-        See basic input variable :ref:`kind <kind>`.
     eink_st_abzuege_params
         See params documentation :ref:`eink_st_abzuege_params <eink_st_abzuege_params>`.
+    anz_erwachsene_tu
+        See func `anz_erwachsene_tu <anz_erwachsene_tu>`.
 
     Returns
     -------
 
     """
     # so far, only the Sonderausgabenpauschale is considered
 
-    if kind:
-        out = 0.0
-    else:
-        out = eink_st_abzuege_params["sonderausgabenpauschbetrag"]["single"]
+    out = (
+        eink_st_abzuege_params["sonderausgabenpauschbetrag"]["single"]
+        * anz_erwachsene_tu
+    )
 
     return float(out)
 
 
-@add_rounding_spec(params_key="eink_st_abzuege")
-def sonderausgaben_betreuung(
+def eink_st_abz_betreuungskost(
     eink_st_abzuege_params: dict,
-    anz_kinder_bis_13_tu: int,
     betreuungskost_m: float,
-    anz_erwachsene_tu: int,
 ) -> float:
-    """Calculate individual sonderausgaben for childcare for childen under 14.
+    """Calculate individual deductable childcare cost for each individual
+    childen under 14.
 
-    We follow 10 Abs.1 Nr. 5 EStG. You can
-    details here https://www.buzer.de/s1.htm?a=10&g=estg.
-    Parameters
     ----------
     betreuungskost_m
         See basic input variable :ref:`betreuungskost_m <betreuungskost_m>`.
-    anz_kinder_bis_13_tu
-        See :func:`anz_kinder_bis_13_tu`.
-    anz_erwachsene_tu
-        See :func:`anz_erwachsene_tu`.
     eink_st_abzuege_params
         See params documentation :ref:`eink_st_abzuege_params <eink_st_abzuege_params>`.
     """
-
-    abziehbare_betreuungskosten = min(
+    out = min(
         12 * betreuungskost_m,
         eink_st_abzuege_params["kinderbetreuungskosten_abz_maximum"],
     )
+    return out
+
+
+@add_rounding_spec(params_key="eink_st_abzuege")
+def sonderausgaben_betreuung_tu(
+    eink_st_abzuege_params: dict,
+    eink_st_abz_betreuungskost_tu: float,
+) -> float:
+    """Calculate sonderausgaben for childcare per tax unit.
```

Comment:
Can you make sure we use `on tax unit level` instead of `per tax unit` in all docstrings (just to be consistent). And similarly get rid of all `per household`